docs/faq/code-analysis/which-metrics-does-codacy-calculate.md

```diff
@@ -112,7 +112,7 @@ Codacy calculates code coverage as follows:
 -   The coverage value for each file is the percentage of coverable lines that are covered by tests in the file
 -   A repository is considered to have acceptable coverage if the average coverage value for the files in the repository is higher than the threshold [**Coverage is under**](../../repositories-configure/adjusting-quality-settings.md#goals)
 -   The coverage variation value of a commit or pull request is the number of percentage points that the coverage value for the file increased or dropped in the commit or pull request
--   The diff coverage of a pull request is the percentage of coverable lines that were added or modified in the pull request that are covered by tests
+-   The diff coverage of a pull request is the percentage of coverable lines added or modified in the pull request that are covered by tests, or `∅` (not applicable) if no coverable lines were changed
```

Comment:
The sentence starts to be a bit long/difficult to me. Not sure if the `, or` could be a note or a sub-bullet? But maybe that is worse 😅 . Either way, it is understandable, just something that starts to be a bit too big for my *sentence comprehension buffer* :D